docs/section22/Rule22-13.md

```diff
@@ -0,0 +1,43 @@
+
+# CHW&CW - Rule 22-13  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-13  
+**Rule Description:** The baseline heat rejection loop shall be an axial-fan open circuit cooling tower.
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each HeatRejection 
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+  - check if heat_rejection fluid_loop is one of the applicable loops: `if heat_rejection.loop.id in heat_rejection_loop_ids: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  ## Rule Logic:  
+
+  - get the fan type: `fan_type = heat_rejection.fan_type`
+  - get the heat rejection type: `heat_rejection_type = heat_rejection.type`
+
+    **Rule Assertion - HeatRejection:**
+
+    - Case 1: If the heat rejection has an axial fan and is an open-circuit cooling tower, PASS: `if ( fan_type == AXIAL ) AND (heat_rejection_type == CLOSED_CIRCUIT_COOLING_TOWER): PASS`
```

Comment:
should say "...heat_rejection_type == **OPEN**_CIRCUIT_COOLING_TOWER". Otherwise looks good.

Reply:
thank you!  Yes, has been changed